src/richie/plugins/slider/models.py

```diff
@@ -0,0 +1,68 @@
+"""
+Slider plugin models
+"""
+
+from django.db import models
+from django.utils.text import Truncator
+from django.utils.translation import gettext_lazy as _
+
+from cms.models.pluginmodel import CMSPlugin
+from filer.fields.image import FilerImageField
+
+
+class Slider(CMSPlugin):
+    """
+    Slide container plugin for Slide item plugins.
+    """
+
+    title = models.CharField(_("title"), max_length=255)
+
+    def __str__(self):
+        return Truncator(self.title).words(6, truncate="...")
```

Comment:
Why do you need to truncate the str representation ?

Reply:
Done